db/migrate/20200311135425_clear_approved_users_from_the_review_queue.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class ClearApprovedUsersFromTheReviewQueue < ActiveRecord::Migration[6.0]
+  def up
+    stuck_user_ids = DB.query_single <<~SQL
+      select u.id
+      FROM users u
+      INNER JOIN reviewables r ON r.target_id = u.id
+      WHERE u.approved = true AND r.type = 'ReviewableUser' AND r.status = #{Reviewable.statuses[:pending]}
+    SQL
+
+    DB.exec(
+      "UPDATE users SET approved = false WHERE id IN (:user_ids)",
+      user_ids: stuck_user_ids
+    )
+
+    system_user = Discourse.system_user
+    ReviewableUser.where(target_id: stuck_user_ids).find_each { |ru| ru.perform(system_user, :approve_user) }
```

Comment:
Even this is unfortunately not going to work in a migration for the long term :(

Is there any way you could use SQL to approve them? If it's only for stuck users it needn't be perfect.